### DISCUSSION TITLE
Value input field with suffix icon

### DISCUSSION THREAD

**Discussion by KateSniegocka** [twilio-labs/paste#2294] [Office hours] [answered]
Currently in Console we have an input field that allows the users to change the default "number input" with buttons to do step +1 or -1 of the number value. 
Developers are currently migrating pages that contains this type of inputs, but it looks like we might not have this variant yet available in Paste?

**Current, Console non-Paste component:**

<img width="512" alt="Screenshot 2022-03-11 at 19 16 25" src="https://user-images.githubusercontent.com/97667802/157936328-5b5a3e3d-2047-447e-b6b5-86112fd7838f.png">

**Potential Paste component:**

![Screenshot 2022-03-11 at 19 07 36](https://user-images.githubusercontent.com/97667802/157936620-740d9137-546c-4e70-b2c1-d8e70c71b981.png)

**Comment by nkrantz** [comment, marked as answer]
Hey @KateSniegocka! You're right, this component isn't available in Paste. It's in our Roadmap to work on it in the future, but I don't have any information about when it'll be available because it's not yet linked to a specific release date. I'd suggest you use the regular number Input component for now. Let me know if that'll work for you!

**Reply by KateSniegocka** [reply]
Hi Nora @nkrantz , thank you very much for the info. We will be using the regular input for now and change it to the new component as soon as it will become available. Just FYI the above example is not the only instance, but it seems to be appearing regularly, another example below. 

<img width="408" alt="Screenshot 2022-03-16 at 12 27 44" src="https://user-images.githubusercontent.com/97667802/158590853-d48efd69-c2fa-4279-ac68-a90aaa99c457.png">

**Reply by nkrantz** [reply]
That sounds good! Thanks for letting us know. 

Also, in case you're interested, [here's some information on why we don't use `input type="number"`](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/) for our inputs and how that pattern can cause inaccessibility.
